### PR TITLE
Added animation library support for Godot 4.0

### PR DESCRIPTION
**Pull request by poohcom1**
Recently I've started using animation libraries in gd4, so I modified the addon to support them. Since it wasn't all that complicated, I thought I'd contribute.

Changes:
- Automatically imports animation names with a slash into the correct animation library
- Added validation for animation names

Looking forward to your feedback!

### CODE REVIEW

File: addons/AsepriteWizard/animation_player/animation_creator.gd [hunk 1]
```diff
@@ -107,16 +107,36 @@ func _configure_animations(target_node: Node, player: AnimationPlayer, content:
 
 func _add_animation_frames(target_node: Node, player: AnimationPlayer, anim_name: String, frames: Array, context: Dictionary, direction = 'forward'):
 	var animation_name = anim_name
+	var library_name = _DEFAULT_ANIMATION_LIBRARY
 	var is_loopable = _config.is_default_animation_loop_enabled()
 
+	var anim_tokens := anim_name.split("/")
+
+	if anim_tokens.size() > 2:
+		push_error("Invalid animation name: %s" % animation_name)
```
Comment (viniciusgerevini) [inline addons/AsepriteWizard/animation_player/animation_creator.gd:116]: Given this is gonna break existing animations which don´t fit the format, I think we can give the option for people to disable it.

You can follow the `is_loopable` example on line 111. It should be a project setting disable by default.

File: addons/AsepriteWizard/animation_player/animation_creator.gd [hunk 2]
```diff
@@ -150,6 +170,13 @@ func _add_animation_frames(target_node: Node, player: AnimationPlayer, anim_name
 	return result_code.SUCCESS
 
 
+const _INVALID_TOKENS := ["/", ":", ",", "["]
```
Comment (viniciusgerevini) [inline addons/AsepriteWizard/animation_player/animation_creator.gd:173]: This looks like dead code. Can you remove it?

**Review by viniciusgerevini — CHANGES_REQUESTED** [review]
This is a pretty good idea. I'll test it a little bit, but it looks good already. Thank you

File: addons/AsepriteWizard/animation_player/animation_creator.gd (hunk 2 above)
Comment (poohcom1) [reply addons/AsepriteWizard/animation_player/animation_creator.gd:177]: Ah, I forgot to actually use the validate function. I've added it to the main importer.

File: addons/AsepriteWizard/animation_player/animation_creator.gd (hunk 1 above)
Comment (poohcom1) [reply addons/AsepriteWizard/animation_player/animation_creator.gd:116]: Any animation name with more than 1 slash is invalid and will cause Godot to error regardless (at least in Godot 4), so I believe this is necessary.

Comment (viniciusgerevini) [reply addons/AsepriteWizard/animation_player/animation_creator.gd:116]: Oh! I see. I wasn't aware of it. Good one. In that case this is also a necessary fix. Thank you.

**Review by viniciusgerevini — APPROVED** [review]
Works like a charm. Thanks for this great contribution.

**Pull request merged**